addons/website_slides/views/slide_channel_views.xml

```diff
@@ -45,23 +45,26 @@
                                 class="oe_stat_button"
                                 groups="website_slides.group_website_slides_officer"
                                 attrs="{'invisible': [('allow_comment', '=', False)]}">
-                                <field name="rating_count" string="Reviews" widget="statinfo"/>
+                                <div class="o_field_widget o_stat_info">
+                                    <span class="o_stat_value"><field name="rating_avg_stars" nolabel="1"/> (<field name="rating_count" nolabel="1"/>)</span>
+                                    <span name="rating_count_label" class="o_stat_text">Reviews</span>
+                                </div>
```

Comment:
well, for me, this is not really understandable.
maybe we should do something like 

```suggestion
                                <div class="o_field_widget o_stat_info">
                                    <span class="o_stat_value"><field name="rating_avg_stars" nolabel="1"/></span>
                                    <span name="rating_count_label" class="o_stat_text"><field name="rating_count" nolabel="1"/> Reviews</span>
                                </div>
```

Reply:
Personally I like it the way it's in the spec ^^
What do you think @sbuhl ?

Reply:
or:
`<div class="o_field_widget o_stat_info">
        <span class="o_stat_value"><field name="rating_avg_stars" nolabel="1"/> /5</span>
                                    <span name="rating_count_label" class="o_stat_text"><field name="rating_count" nolabel="1"/> Reviews</span>
                                </div>`
For this result:
<img width="168" alt="Screenshot 2020-06-11 at 16 35 40" src="https://user-images.githubusercontent.com/505562/84399832-9b612080-ac01-11ea-9495-e6f6b82285c7.png">